pom.xml

```diff
@@ -1545,6 +1547,7 @@
         <hbase.profile>2.1</hbase.profile>
         <hbase.compat.version>2.1.6</hbase.compat.version>
         <hbase.version>${hbase-2.1.runtime.version}</hbase.version>
+        <hadoop.version>3.0.3</hadoop.version>
```

Comment:
While confirming `hadoop-three.version` on branch-2.1, when I saw default value of `${hbase-2.1.runtime.version}` as `2.1.10` I was surprised to see only 2.1.10 RC tags available on HBase but release tag `rel/2.1.10` is missing!